packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -121,6 +121,33 @@ export class MonacoEditor extends MonacoEditorServices implements TextEditor {
         return this.document.setEncoding(encoding, mode);
     }
 
+    findMatches(options: FindMatchesOptions): FindMatch[] {
+        const model = this.editor.getModel();
+        if (!model) {
+            return [];
+        }
+        const results: monaco.editor.FindMatch[] = model.findMatches(
+            options.searchString,
+            false,
+            options.isRegex,
+            options.matchCase,
+            // eslint-disable-next-line no-null/no-null
+            options.matchWholeWord ? options.searchString : null,
```

Comment:
The parameter name is `wordSeparators` here, I don't think passing `options.searchString` is the way to go.

Reply:
Updated the PR to address this :) Thanks for the catch.